apps/notifications/src/panel/templates/block-user.jsx

```diff
@@ -162,15 +162,28 @@ export class UserBlock extends Component {
 			);
 		}
 
+		const userId = this.props.block?.meta?.ids?.user;
```

Comment:
Good thinking! That's right, we don't in fact need the username here, the user ID works just fine 🙌